hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestPartialUpdateAvroPayload.scala

```diff
@@ -66,8 +66,14 @@ class TestPartialUpdateAvroPayload extends HoodieClientTestBase {
   }
 
   @ParameterizedTest
-  @EnumSource(classOf[HoodieTableType])
-  def testPartialUpdatesAvroPayloadPrecombine(hoodieTableType: HoodieTableType): Unit = {
+  @CsvSource(Array(
```

Comment:
Similar here.

Reply:
I didn't notice the test you mentioned. I can change that in a subsequent PR. Without the fix, this test will fail for the last case ("MERGE_ON_READ", true).